lms/djangoapps/courseware/tests/test_courses.py

```diff
@@ -457,3 +458,21 @@ def test_completion_does_not_count_empty_sequentials(self):
         assignments = get_course_assignments(course.location.context_key, self.user, None)
         assert len(assignments) == 1
         assert not assignments[0].complete
+
+    def test_completion_does_not_treat_unreleased_as_complete(self):
+        """
+        Test that unreleased assignments are not treated as complete.
+        """
+        course = CourseFactory()
+        chapter = ItemFactory(parent=course, category='chapter', graded=True,
+                              start=datetime.datetime.now() + datetime.timedelta(hours=1))
```

Comment:
I do like the more explicit start date, rather than relying on the default 2030 start date. :+1: